hbase-server/src/main/java/org/apache/hadoop/hbase/regionserver/LeaseManager.java

```diff
@@ -216,7 +216,7 @@ public String getName() {
   }
 
   /** This class tracks a single Lease. */
-  static class Lease implements Delayed {
+  protected static class Lease implements Delayed {
```

Comment:
Not sure if there's a better way to do this, but I needed to expose this to implement [this](https://github.com/apache/hbase/pull/6055/files#diff-73a63532c8c4d8c7bb2eb4663fb7303ddf1ffe9f1f508221f566a8c156d70b11R155) method to track the leases created

Reply:
Isn't `package protected` is a super-set of `protected` permissions? The `LeaseManager$Lease` class should already be accessible to anything in the `o.a.h.h.regionserver` package.

Reply:
Unfortunately, `NoopOperationQuota` is also protected in the same way LeaseManager is. Originally, this was the reason I placed the test under the `quotas` package. I considered creating a test OperationQuota class that is a 1:1 copy of `NoopOperationQuota`, but I didn't love that. Though I'm happy to go that route if it's preferred

Reply:
Alternatively, I considered tracking the # of scanners created, but the code in RSRpcServices makes it tricky to do. All the relevant methods are private, and I was very reluctant to expose them.

Reply:
I have an idea, iterating on this now.